### ISSUE TITLE
[Platform][There are no documented limits for Input.Number max value]

### ISSUE THREAD

**Issue by tacri** [microsoft/AdaptiveCards#5404]
# Platform

What platform is your issue or question related to? (Delete other platforms).

- [x] .NET HTML
- [x] .NET WPF
- [x] Android
- [x] iOS
- [x] JavaScript
- [x] UWP

# Author or host

Are you an author (like sending something to Outlook), or a host that is rendering your own cards? If host, please specify the corresponding host (app) name. 

Teams app

If you're an author, who are you sending cards to?

# Version of SDK

What version are you using? Ex: NuGet 1.0.2, or latest main, etc...

# Details

Explain your issue. If you just have a question, please post [on StackOverflow instead](https://stackoverflow.com/questions/tagged/adaptive-cards).

In Teams the input.number is not allowing more than a 10 digit number. This was an arbitrary value picked by our channel. The spec doesn't define the max value allowed for this field. Without a known, well defined value there are different expectations from users on what should work resulting in customer escalation.

**Comment by paulcam206** [comment]
We should document our limits here.

**Comment by jonmill** [comment]
tldr; we don't have set limits for max today, so we should do two things:
1) Add a min / max value across all platforms (which is the same value)
2) Document this number

**Comment by aishwarya88057657** [comment]
Hi @jonmill, @RebeccaAnne , @paulcam206
Is there any update on this issue?

We are also facing this issue
**Short Description:** Input.Number is not allowing more than 10 numbers
**Description:**
In mobile number scenario if user enter mobile number more than 10 digit then at the validation time we are trying to show that number on card but Adaptive card not allowing to display that number and throwing exception as number consist of more than 10 digit.
